spec/requests/swagger_spec.rb

```diff
@@ -0,0 +1,454 @@
+require 'rails_helper'
+
+RSpec.describe 'Swagger spec', type: :apivore, order: :defined do
```

Comment:
I tried that. Putting them all in here let's me do the trick at the bottom where it fails if there is an API endpoint I have described but I haven't checked. See here https://github.com/westfieldlabs/apivore for details